client/me/concierge/shared/closure-notice.js

```diff
@@ -29,7 +29,7 @@ const ClosureNotice = ( { closesAt, displayAt, holidayName, reopensAt } ) => {
 
 	if ( currentDate.isBefore( closesAt ) ) {
 		message = translate(
-			'{{strong}}Note:{{/strong}} Support sessions will be closed for %(holidayName)s from %(closesAt)s until %(reopensAt)s. ' +
+			'{{strong}}Note:{{/strong}} Quick Start sessions will be closed for %(holidayName)s from %(closesAt)s until %(reopensAt)s. ' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( '{{strong}}Note:{{/strong}} Support sessions will be closed for %(holidayName)s from %(closesAt)s until %(reopensAt)s. If you need to get in touch with us, you’ll be able to {{link}}submit a support request{{/link}} and we’ll get to it as fast as we can. Thank you!' )` *ES Score: 8*